src/scripts/background/types.ts

```diff
@@ -71,6 +78,8 @@ export const DEFAULT_KEYBINDS: Keybinds = {
   'increase-current-time-large': 'Shift+L',
   'decrease-current-time': 'j',
   'decrease-current-time-large': 'Shift+J',
+  'seek-forward-one-frame': '.',
```

Comment:
`Ctrl` + `Alt` + `->`
`Ctrl` + `Alt` + `<-`
?

Reply:
Might leave it as is as it's the default YouTube keybinds